src/bootchooser.c

```diff
@@ -44,27 +44,36 @@ static gboolean bootchooser_order_primay(RaucSlot *slot, GString **value) {
 	return TRUE;
 }
 
+typedef struct {
+	guint32 prio;
+	guint32 attempts;
+} BareboxSlotState;
+
+#define BOOTSTATE_PREFIX "bootstate"
+
 #if 0
```

Comment:
According to the commit message, the function should be reenabled.

Reply:
This would require squashing commits if we intend to keep the commit compilable.

Reply:
Hmm, OK. Then explain that in the commit message.